spec/features/splashscreen_index_spec.rb

```diff
@@ -0,0 +1,21 @@
+require 'rails_helper'
+
+RSpec.feature 'Splashscreen', type: :feature, js: true do
+  scenario 'Visiting the root path' do
+    visit root_path
+
+    expect(page).to have_content('Grey Budget App')
+  end
+
+  scenario 'Click sign in' do
+    visit root_path
+    click_link 'Sign in'
+    expect(page).to have_current_path('/users/sign_in')
+  end
+
+  scenario 'Click sign up' do
+    visit root_path
+    click_link 'Sign up'
+    expect(page).to have_current_path('/users/sign_up')
+  end
+end
```

Comment:
- [ ] I will encourage you to fix the tests that are failing as pointed out in the image below. 
![image](https://github.com/DebyGrey/budgetapp/assets/86200356/2f723a71-b382-47b3-85e0-d0ae8ad0fca4)